src/editor.js

```diff
@@ -261,34 +283,86 @@ const ContentPanel = ({
   content,
   isDisplayed,
   isFilesPanelDisplayed,
+  isTemplatePanelDisplayed,
+  templates,
 }) => html`
   <div class=${n('content')} ?hidden=${!isDisplayed}>
-    ${ContentToolbar({isFilesPanelDisplayed})}
+    ${ContentToolbar({isFilesPanelDisplayed, isTemplatePanelDisplayed})}
     <!--
         Default Content to load on the server and then populate codemirror on
         the client.
         codeMirrorElement is a promise resolved by codemirror(), hence the
         until directive. Once resolved, content can be empty.
       -->
+    ${TemplatesPanel({isTemplatePanelDisplayed, templates})}
     ${until(codeMirrorElement || Textarea({content}))}
   </div>
 `;
+const dispatchSelectTemplate = redispatchAs(g('select-template'));
+
+const TemplatesPanel = ({isTemplatePanelDisplayed, templates}) =>
+  isTemplatePanelDisplayed
```

Comment:
Please add a note here that we're preventing render when not displayed due to video playback performance.